src/ui/components/App.tsx

```diff
@@ -79,8 +78,23 @@ function setTelemetryContext(userId: string | undefined, userEmail: string | und
 function App({ theme, recordingId, modal, updateNarrowMode }: AppProps) {
   const auth = useAuth0();
   const { claims } = useToken();
+
   setTelemetryContext(claims?.hasura.userId, auth.user?.email);
   const { loading } = hooks.useMaybeClaimInvite();
+  const [fontLoading, setFontLoading] = useState(true);
```

Comment:
This is a good candidate to refactor into a small custom hook to help keep the App renderer readable.

```js
const useFontLoader = () => {
  const [fontLoading, setFontLoading] = useState(true);
  useEffect(() => {
    var font = new FontFaceObserver("Material Icons");

    font.load().then(() => {
      setFontLoading(false);
    });

    // FontFaceObserver doesn't work in e2e tests.
    if (isTest()) {
      setFontLoading(false);
    }
  }, []);

  return fontLoading;
}

// ...

const fontLoading = useFontLoading();
```